tests/ui-toml/excessive_nesting/auxiliary/proc_macros.rs

```diff
@@ -0,0 +1,476 @@
+//@compile-flags: --emit=link
+//@no-prefer-dynamic
+
+// NOTE: Copied from `ui/auxiliary/proc_macros.rs`, couldn't get `../` to work for some reason
```

Comment:
The path starts at `excessive_nesting/auxiliary`, and `excessive_nesting/auxiliary/..` is considered not to exist on linux (but does exist on windows where paths are first resolved lexically)

A copy is unfortunate but it's something we can look to fix separately